### PR TITLE
docs: refactor for self-hosted Palette DOC-465

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR refactors self-hosted Palette instructions to mirror the information architecture of Palette VerteX.

The following pages contain new content:
- [Backup and Restore](https://deploy-preview-1597--docs-spectrocloud.netlify.app/enterprise-version/system-management/backup-restore)
- [System Management Index](https://deploy-preview-1597--docs-spectrocloud.netlify.app/enterprise-version/system-management)
- [VMware Permissions](https://deploy-preview-1597--docs-spectrocloud.netlify.app/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements)

## Review Changes

💻 [Preview URL](https://deploy-preview-1597--docs-spectrocloud.netlify.app/enterprise-version/)

🎫 [DOC-465](https://spectrocloud.atlassian.net/browse/DOC-465)

### CODE REVIEW

**Review by ritawatson — APPROVED** [review]
Approving this, but there are several things to address. 
As instructed, I focussed on the two files listed in the PR description (Backup/Restore and System Management) but noticed a few fixes to make in a spot check of other files.

File: docs/deprecated/enterprise-version/enterprise-version.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Install Palette onto a Kubernetes cluster using a Helm Chart. Review the [Helm Chart Mode](deploying-palette-with-helm.md) guide to learn more.
+
+
+## Airgap Install
+
+Palette can be installed in a VMware environment without internet access, known as an air gap installation, requiring pre-download of platform manifests, required platform packages, container images for core components, third-party dependencies, and Palette Packs, all sourced from a private rather than the default public Palette repository.
+
+## Download Palette Installer
+
+To request the Palette Self-hosted installer image, please contact our support team by sending an email to support@spectrocloud.com. Kindly provide the following information in your email:
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/deprecated/enterprise-version/enterprise-version.md:34]: Nice catch @ritawatson

File: docs/docs-content/enterprise-version/system-management/backup-restore.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+1. Log into the Palette system console as an administrator. Refer to the [Access the System Console](../system-management/system-management.md#access-the-system-console) section for more information.
+
+
+2. From the left **Main Menu**, select **Administration**.
+
+
+3. Click on the **Backup/Restore** tab.
+
+
+4. Select the **FTP** and fill out the following fields:
```
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/system-management/backup-restore.md:42]:
```suggestion
4. Select the **FTP** tab and fill out the following fields:
```

File: docs/docs-content/enterprise-version/system-management/backup-restore.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    ]
+}
+```
+
+- Credentials to the IAM user. You need the AWS access key ID and the AWS secret access key.
+
+
+### Instructions
+
+1. Log into the Palette system console as an administrator. Refer to the [Access the System Console](../system-management/system-management.md#access-the-system-console) section for more information.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/enterprise-version/system-management/backup-restore.md:120]: Looking at it now, I don't think we need either as only admins can log into the system console.

File: docs/docs-content/enterprise-version/system-management/backup-restore.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+
+2. From the left **Main Menu**, select **Administration**.
+
+
+3. Click on the **Backup/Restore** tab.
+
+
+4. Select the **FTP** and fill out the following fields:
```
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/system-management/backup-restore.md:129]:
```suggestion
4. Select the **FTP**tab and fill out the following fields:
```

File: docs/docs-content/enterprise-version/system-management/backup-restore.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+### Instructions
+
+1. Log into the Palette system console as an administrator. Refer to the [Access the System Console](../system-management/system-management.md#access-the-system-console) section for more information.
+
+
+2. From the left **Main Menu**, select **Administration**.
+
+
+3. Click on the **Backup/Restore** tab.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/enterprise-version/system-management/backup-restore.md:126]: You're looking at the Palette UI, not the Palette system console. Remember, system console is a the `/system` endpoint.

File: docs/docs-content/enterprise-version/system-management/system-management.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+The system console enables you to complete the initial setup and onboarding and manage the overall Palette environment.
+
+### Access the System Console
+
+You can access the system console by visiting the IP address or the custom domain name assigned to your Palette cluster and appending the `/system` path to the URL. For example, if your Palette cluster is hosted at `https://palette.abc.com`, you can access the system console at `https://palette.abc.com/system`.
+
+
+## Administration and Management
+
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/enterprise-version/system-management/system-management.md:33]: That's exactly why.

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+
+The permissions listed in this page are also needed for deploying a Private Cloud Gateway (PCG), and workload cluster in vSphere through Palette.
+:::
+
+
+## Create Required Roles
+
+Palette requires two custom roles to be created in vSphere before the installation. Refer to the [Create a Custom Role](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html?hWord=N4IghgNiBcIE4HsIFMDOIC+Q) guide if you need help creating a custom role in vSphere. The required custom roles are:
+
+* A root-level role with access to higher-level vSphere objects. This role is referred to as the *spectro root role*. Check out to the [Root-Level Role Privileges](#root-level-role-privileges) table for the list of privileges required for the root-level role.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:33]:
```suggestion
* A root-level role with access to higher-level vSphere objects. This role is referred to as the *spectro root role*. Check out the [Root-Level Role Privileges](#root-level-role-privileges) table for the list of privileges required for the root-level role.
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+
+The user account you use to deploy Palette must have access to both roles. Each vSphere object required by Palette must have a [Permission](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-4B47F690-72E7-4861-A299-9195B9C52E71.html) entry for the respective Spectro role. The following tables list the privileges required for the each custom role. 
+
+
+
+
+:::info
+
+For an-idepth overview of vSphere authorization and permissions overview, check out the [Understanding Authorization in vSphere](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-74F53189-EF41-4AC1-A78E-D25621855800.html) resource.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:45]: Is it an "in-depth" (typo: an-idepth) or an "overview"? Maybe the following is intended?

Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:45]:
```suggestion
For an in-depth explanation of vSphere authorization and permissions, check out the [Understanding Authorization in vSphere](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-74F53189-EF41-4AC1-A78E-D25621855800.html) resource.
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+:::caution
+
+If the network is a Distributed Port Group under a vSphere Distributed Switch (VDS), *ReadOnly* access to the VDS without “Propagate to children” is required.
+
+:::
+
+
+### Spectro Role Privileges
+
+The spectro role privileges listed in the following table must be applied to following vSphere objects you intended to use for the Palette install. A separate table lists Spectro role privileges for VMs by category.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:121]:
```suggestion

As listed in the table, apply spectro role privileges to vSphere objects you intend to use for Palette installation. A separate table lists Spectro role privileges for VMs by category. 
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+| **Host**              | Local operations: Reconfigure VM        |
+| **Network**           | Assign network                              |
+| **Resource**          | Apply recommendation<br />Assign VM to resource pool<br />Migrate powered off VM<br />Migrate powered on VM<br />Query vMotion |
+| **Sessions**          | Validate sessions                           |
+| **Storage policies**  | View access for VM storage policies is required.<br />Ensure ``StorageProfile.View`` is available. |
+| **spectro-templates** | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.                                  |
+| **Storage views**     | View                 |
+| **Tasks**             | Create task<br />Update task                |
+| **vApp**              | Import<br />View OVF environment<br />Configure vAPP application<br />Configure vApp instance |
+| **vSphere tagging**   | Assign or Unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:147]:
```suggestion
| **vSphere tagging**   | Assign or unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 11]
```diff
@@ -0,0 +1,300 @@
...
+| **Host**              | Local operations: Reconfigure VM        |
+| **Network**           | Assign network                              |
+| **Resource**          | Apply recommendation<br />Assign VM to resource pool<br />Migrate powered off VM<br />Migrate powered on VM<br />Query vMotion |
+| **Profile-driven storage** | Profile-driven storage view            |
+| **Sessions**          | Validate session                            |
+| **spectro-templates** | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.                                    |
+| **Storage views**     | Configure service<br />View                 |
+| **Tasks**             | Create task<br />Update task                |
+| **vApp**              | Import<br />View OVF environment<br />Configure vAPP applications<br />Configure vApp instances |
+| **vSphere tagging**   | Assign or Unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:186]:
```suggestion
| **vSphere tagging**   | Assign or unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 12]
```diff
@@ -0,0 +1,300 @@
...
+| **Host**              | Local operations: Reconfigure VM            |
+| **Network**           | Assign network                              |
+| **Profile-driven storage** | Profile-driven storage view            |
+| **Resource**          | Apply recommendation<br />Assign VM to resource pool<br />Migrate powered off VM<br />Migrate powered on VM<br />Query vMotion |
+| **Sessions**          | Validate session                            |
+| **spectro-templates** | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.    |
+| **Storage views**     | View                                        |
+| **Tasks**             | Create task<br />Update task                |
+| **vApp**              | Import<br />View OVF environment<br />Configure vAPP applications<br />Configure vApp instances |
+| **vSphere tagging**   | Assign or Unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:226]:
```suggestion
| **vSphere tagging**   | Assign or unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 13]
```diff
@@ -0,0 +1,300 @@
...
+
+
+
+The following table lists spectro role privileges for VMs by category. All privileges are for the vSphere object, Virtual Machines.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:230]: The last sentence doesn't make sense to me: "All privileges are for the vSphere object, Virtual Machines."

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 14]
```diff
@@ -0,0 +1,300 @@
...
+
+</TabItem>
+
+
+</Tabs>
+
+
+
+## Zone Tagging
+You can use tags to create node zones and regions for your Kubernetes clusters. The node zones and regions can be used to dynamically place Kubernetes workloads and achiveve higher availability. Kubernetes nodes inherit the zone and region tags as [Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). Kubernetes workloads can use the node labels to ensure that the workloads are deployed to the correct zone and region.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:252]:
```suggestion
You can use tags to create node zones and regions for your Kubernetes clusters. The node zones and regions can be used to dynamically place Kubernetes workloads and achieve higher availability. Kubernetes nodes inherit the zone and region tags as [Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). Kubernetes workloads can use the node labels to ensure that the workloads are deployed to the correct zone and region.
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 15]
```diff
@@ -0,0 +1,300 @@
...
+
+To learn more about node zones and regions, refer to the [Node Zones/Regions Topology](https://cloud-provider-vsphere.sigs.k8s.io/cloud_provider_interface.html) section of the Cloud Provider Interface documentation.
+
+:::
+
+
+Zone tagging is required to install Palette and is helpful for Kubernetes workloads deployed in vSphere clusters through Palette if they have persistent storage needs. Use vSphere tags on data centers and compute clusters to create distinct zones in your environment. You can use vSphere [Tag Categories and Tags](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html) to create zones in your vSphere environment and assign them to vSphere objects.
+
+
+The zone tags you assign to your vSphere objects, such as a datacenter and clusters are applied to the Kubernetes nodes you deploy through Palette into your vSphere environment. Kubernetes clusters deployed to other infrastructure providers, such as public cloud may have other native mechanisms for auto discovery of zones.
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:274]: We don't have "data center" on our word list and should add it. There's a mix of single-word and two-word occurrences in this section. I think we decided on two words, as it appears that way in our nav pane. Let's confirm with each other in our team meeting, and I'll add it to the word list.

**Review by ritawatson — APPROVED** [review]
There are a few minor things to address.

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md (hunk 13 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:230]:
```suggestion
The following table lists spectro role privileges for the vSphere object, Virtual Machine. 
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 16]
```diff
@@ -0,0 +1,300 @@
...
+  
+For example, assume a vCenter environment contains three compute clusters, cluster-1, cluster-2, and cluster-3. To support this environment you create the tag categories `k8s-region` and `k8s-zone`. The `k8s-region` is assigned to the data center and the `k8s-zone` tag is assigned to the compute clusters.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:276]:
```suggestion
For example, assume a vCenter environment contains three compute clusters, cluster-1, cluster-2, and cluster-3. To support this environment you create the tag categories `k8s-region` and `k8s-zone`. The `k8s-region` is assigned to the datacenter, and the `k8s-zone` tag is assigned to the compute clusters.
```

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md [hunk 17]
```diff
@@ -0,0 +1,300 @@
...
+
+The following table lists the tag values for the data center and compute clusters.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md:278]:
```suggestion
The following table lists the tag values for the vSphere objects, datacenter, and compute clusters.
```

**Comment by karl-cardenas-coding** [comment]
This PR  #1646  adds the changes to the 3.4 version branch.

**Pull request merged**